documentation/quarkus/guide-api-first.asciidoc

```diff
@@ -0,0 +1,153 @@
+:toc: macro
+toc::[]
+
+= API first development guide
+
+Cloud native promotes the use of microservices, which are loosely coupled and self-contained. These services communicate with each other using a well-defined interface or API, and no consumer needs to be aware of any implementation details of the provider. There could even be multiple providers of the same API, or we can decide to swap existing implementation for a new one, without disrupting our clients - all because we adhere to a well defined API. 
+This guide focuses on HTTP interfaces following RESTful design principles. 
+
+== OpenAPI
```

Comment:
There is already a guide about OpenAPI, so I think this is duplicate and can be removed. Or just link to the other guide